mobile/src/common/rounded-avatar.tsx

```diff
@@ -12,14 +12,14 @@ export interface Props {
     innerBorderWidth: number;
 }
 
-export class RoundedAvatar extends Component<Props> {
+export class RoundedAvatar extends Component<RoundedAvatarProps> {
     public render() {
         const { outerBorderColor, innerBorderColor, photoBase64, avatarSize, outerBorderWidth, innerBorderWidth } = this.props;
         const imageSize = avatarSize - outerBorderWidth;
 
         return (
             <View style={{ height: avatarSize, width: avatarSize, justifyContent: 'center', alignItems: 'center', borderRadius: avatarSize * 0.5, borderColor: outerBorderColor, borderWidth: outerBorderWidth }}>
-                <Image source={{uri: photoBase64}} style={{ height: imageSize, width: imageSize, borderRadius: imageSize * 0.5, borderColor: innerBorderColor, borderWidth: innerBorderWidth }} />
+                <Image source={{uri: 'data:image/png;base64,' + photoBase64}} style={{ height: imageSize, width: imageSize, borderRadius: imageSize * 0.5, borderColor: innerBorderColor, borderWidth: innerBorderWidth }} />
```

Comment:
I thought that mime type should be provided by the parent component, not hardcoded. Server API returns that as a separate field in Photo model

Reply:
ok, missed this attribute, it will be used